crud/src/main/java/com/redhat/lightblue/rest/crud/health/CrudCheckResponse.java

```diff
@@ -27,9 +27,13 @@ public JsonNode toJson() {
 
             if (result.getValue().getDetails() != null) {
               for (Map.Entry<String, Object> entry : result.getValue().getDetails().entrySet()) {
-                  if(null != entry.getValue()) {
-                      resultResponse.put(entry.getKey(), entry.getValue().toString());
+                if(null != entry.getValue()) {
+                  if(entry.getValue() instanceof ObjectNode) {
+                    resultResponse.put(entry.getKey(), (ObjectNode)entry.getValue());
+                  } else {
+                    resultResponse.put(entry.getKey(), entry.getValue().toString());
```

Comment:
is just putting getValue() enough for all types and then we can let the serialization configuration figure out how to serialize?

Reply:
Yes